browser/components/CodeEditor.js

```diff
@@ -623,6 +627,10 @@ export default class CodeEditor extends React.Component {
 
     spellcheck.setLanguage(null, spellcheck.SPELLCHECK_DISABLED)
     eventEmitter.off('code:format-table', this.formatTable)
+
+    // "Remember" top line number (property of grand-parent component).
+    const currentLineAtTop = this.editor.lineAtHeight(this.editor.getScrollInfo().top, 'local') + 1
+    this.props.topLine.number = currentLineAtTop
```

Comment:
Modifying props is consider as anti-pattern for React. Can you do it some otherway? maybe notify parent to update instead of directly modifying like this